src/_pytest/hookspec.py

```diff
@@ -56,7 +56,10 @@ def pytest_addhooks(pluginmanager: "PytestPluginManager") -> None:
     """Called at plugin registration time to allow adding new hooks via a call to
     ``pluginmanager.add_hookspecs(module_or_class, prefix)``.
 
-    :param pytest.PytestPluginManager pluginmanager: The pytest plugin manager.
+    :param pluginmanager: The pytest plugin manager.
+
+    If a conftest file implements this hook, it will be called immediately
```

Comment:
Really like the new sections!

How about making them more prominent by adding a quick title, something like:

```suggestion
    **Note on conftests**
    
    If a conftest file implements this hook, it will be called immediately
```

This makes them standout and something to look for when browsing the hook documentation.